ocs_ci/ocs/resources/ocs.py

```diff
@@ -224,7 +228,20 @@ def get_ocs_csv():
         subscription_plan_approval=subscription_plan_approval,
     )
     channel = config.DEPLOYMENT.get("ocs_csv_channel")
-    ocs_csv_name = ocs_package_manifest.get_current_csv(channel=channel)
+    ocs_csv_name = None
+    # OCS CSV is extracted from the available CSVs in cluster namespace
+    # for Openshift dedicated platform
+    if config.ENV_DATA["platform"].lower() == "openshiftdedicated":
+        ocp_cluster = OCP(
+            namespace=config.ENV_DATA["cluster_namespace"], kind="", resource_name="csv"
```

Comment:
should not this be changed to:
```suggestion
            namespace=config.ENV_DATA["cluster_namespace"], kind="csv"
```
As you would like to iterate over the CSV objects?

This you should get all kind of csv, and then you can iterate over them.

Just a side note:
When the upgrade happens, you can have 2 CSVs at the same time, one is in replacing state and second one in installing.  But I do not expect that you will somehow test upgrade of OCS on openshift dedicated, right?

Reply:
Fixed.
Not expecting to test upgrade on Openshift dedicated right now.